components/core/NewWebsitePrototypeHeader.js

```diff
@@ -15,7 +15,7 @@ const STYLES_CONTAINER = css`
   position: -webkit-sticky;
   position: sticky;
   top: 0;
-  z-index: 3;
+  z-index: 42;
```

Comment:
Perhaps a typo? 42 -> 4

Reply:
no I was just picking a big number I like to make sure it stays on top of the page elements. Sorry for the confusion tho! I can change it to 20 or something next time :]

Reply:
@tarafanlin there is an order to the chaos, please consider adding this to `~/common/constants` and making sure its on a scale relative to other entities on the screen.

Reply:
just saw the zIndex constants, will adapt those

Reply:
Ahh got it!